python/tank/context.py

```diff
@@ -899,22 +901,40 @@ def serialize(context):
         "additional_entities": context.additional_entities,
         "_pc_path": context.tank.pipeline_configuration.get_path()
     }
+
+    from tank_vendor import shotgun_authentication as sg_auth
+
+    # If there is a current user.
+    user = get_current_user()
+    if user:
+        # We should serialize it as well.
+        data["current_user"] = sg_auth.serialize_user(user)
```

Comment:
note that the `_pc_path` key is named in such a way to indicate that it is special and not part of the std context varialbes. Would be nice if we had `_current_user` as well to follow that pattern.